### DISCUSSION TITLE
Problems with the dpgen init process

### DISCUSSION THREAD

**Discussion by Lweiweij** [deepmodeling/dpgen#1073] [Q&A] [answered]
When I run “dpgen init_bulk ch4.json" for the examples of source code , the following error occurs. How do I fix this ? Thank you for your advice!
![image](https://user-images.githubusercontent.com/114126058/206858986-2ace4e47-6664-417d-bada-e4238a5936b3.png)

**Comment by cancer521** [comment]
I alseo has the same error

**Comment by wanghan-iapcm** [comment]
@HuangJiameng Could you please take a look? It looks like something is not compatible to the latest pymatgen.

**Comment by HuangJiameng** [comment]
I have reproduced the problem and found that although the POSCAR is available, the latest `pymatgen` can not recognize it. @ZLI-afk Could you help take a look? Thanks.

**Comment by cancer521** [comment, marked as answer]
pip uninstall pymatgen   
pip3 install pymatgen==2022.10.20
after install，You should look for dependency issues,Follow the prompts to download the appropriate version of pymatgen.
this all

**Reply by Lweiweij** [reply]
Thank you for your answer, I have solved the problem！
